frontend/controller/actions/identity.js

```diff
@@ -527,16 +532,19 @@ export default (sbp('sbp/selectors/register', {
     await sbp('gi.actions/chatroom/join', {
       ...omit(params, ['options', 'contractID', 'data', 'hooks']),
       contractID: message.contractID(),
-      data: {}
+      // 'undefined' is for ourselves
+      data: { memberID: [undefined, ...partnerIDs] }
```

Comment:
Weirdness: why are we putting `undefined` here instead of `identityContractID`?

Request: put `identityContractID` here and remove the weird `x == null` code in `'gi.actions/chatroom/join'`.

Reply:
Sure. It was here mostly because then `{ memberID: cID }` over there will be `{ memberID: 'foo' }` for others and `{}` for ourselves.

Reply:
The `x == null` is needed to set our identity contractID when not given, which is supported by the contract, is less error prone and makes for smaller messages. However, I simplified the code to make this check only once (instead of twice in `userID` & `userIDs`)